lib/replacer.js

```diff
@@ -1,5 +1,7 @@
 module.exports = function (cow, variables) {
 	var eyes = escapeRe(variables.eyes);
+	var eyeL = eyes.substr(0,1);
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr

<img width="687" alt="screen shot 2018-11-02 at 16 07 23" src="https://user-images.githubusercontent.com/680284/47923370-84047200-deb9-11e8-92b3-c34edec4b096.png">

Can you please use `eyeL = eyes.charAt(0);` and `eyeR = eyes.charAt(1);` it's more readable.